webpack-utils/TranslationsWebpackPlugin.js

```diff
@@ -0,0 +1,14 @@
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+const mergeDeep = require('merge-deep')
+
+const defaultTranslations = require('../src/translations/en.json')
+
+module.exports = () =>
+  new CopyWebpackPlugin([
+    {
+      from: './src/translations/**/*.json',
+      to: './translations/[name].json',
+      transform: content =>
+        JSON.stringify(mergeDeep(defaultTranslations, JSON.parse(content)))
```

Comment:
Here we merge translation files such that we default to english for any locale missing a particular translation value